src/components/TalkToUs/LeaveMessage/index.js

```diff
@@ -43,7 +46,7 @@ const Callback = () => {
     [intl.locale]
   )
 
-  const { formFields, formStatus, handleSubmit, disabled } = useForm(
+  const { formFields, formStatus, handleSubmit, disabled, isLoading } = useForm(
```

Comment:
Use `formStatus` instead of `isLoading`. Please also help change that instead of declare status constant strings in this file:
```js
const SUBMITTED = "submitted"
const FAILED = "failed"
```
export these constants in `useForm.js` and import them for comparison.

Reply:
Done.